en/docs/design-your-project/index.md

```diff
@@ -1,4 +1,4 @@
-# Architecture Diagram
+# Design your project
```

Comment:
Instead of using `your` in titles and url paths shall we use `the`?

Reply:
Updated.